frontend/src/metabase-lib/lib/metadata/Database.ts

```diff
@@ -35,7 +37,11 @@ export default class Database extends Base {
   /**
    * @param {SchemaName} [schemaName]
    */
-  schema(schemaName) {
+  schema(schemaName: string | undefined | null): Schema | undefined {
```

Comment:
There were a few places where we were calling this method with a potentially null or undefined schemaName, so I figured adding this would make using the method a little easier.